pkg/portal/cluster/prometheusalerts_test.go

```diff
@@ -0,0 +1,113 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/go-test/deep"
+	"github.com/golang/mock/gomock"
+	"github.com/prometheus/common/model"
+
+	mock_alertmanager "github.com/Azure/ARO-RP/pkg/util/mocks/alertmanager"
+	testlog "github.com/Azure/ARO-RP/test/util/log"
+)
+
+func TestGetOpenShiftFiringAlerts(t *testing.T) {
+	ctx := context.Background()
+	_, log := testlog.New()
+
+	for _, tt := range []struct {
+		name          string
+		returedData   []model.Alert
+		expected      []FiringAlert
+		errorExpected error
+	}{
+		{
+			name: "return firing alert",
+			returedData: []model.Alert{
+				{
+					Labels: model.LabelSet{
+						"alertname": "Firing Alert 1",
+						"namespace": "openshift-apiserver",
+					},
+					StartsAt: time.Now().Add(-1),
+				},
+				{
+					Labels: model.LabelSet{
```

Comment:
I would consider making a new separate test for "don't return resolved alert". Alternatively, maybe it would make sense to test all this in a single test (pass in `Firing Alert 1`, `Resolved Alert 1`, and other namespace alert, expect only `Firing Alert 1` to be returned in `alerts`).